tests/unit/components/personal-survey-test.js

```diff
@@ -62,8 +62,8 @@ module('Unit | Components | personal-survey', function (hooks) {
     storage.saveUser(user)
   }
 
-  function answerSessionQuestion(id) {
-    personalSurvey.pilasBloquesAnalytics.newAnswer(makeAnswer(id))
+  async function answerSessionQuestion(id) {
+    await personalSurvey.pilasBloquesAnalytics.newAnswer(makeAnswer(id))
   }
```

Comment:
Si hacemos esto, falta clavar el await en el test de la línea 27: 'Should not ask already session answered questions'